### DISCUSSION TITLE
Entity Hider Presets

### DISCUSSION THREAD

**Discussion by josephsam2** [runelite/runelite#15984] [Ideas]
While in the bloat room at TOB, I use hide local player, hide others, and hide friends (in case they are in the raid with me) to make it easier to avoid falling hands, but only after the first down has started. This preset would also be useful in group raids at wardens p4 for dodging lightning. In Sote at TOB, I like to hide NPCs after hammering so I can see the orbs traveling through sote. For p5 nightmare its nice to hide local player to dodge the portals. for Theiving, it is useful to hide others and friends chat if using theiving host or splash worlds. I'm sure there are many other stressful PvM situations where a preset easily turned on and off would be useful.

What would be some other useful presets?

**Comment by antero111** [comment]
You could make these presets with [Plugin Presets](https://runelite.net/plugin-hub/show/plugin-presets) hub plugin. Make presets for each phase (or skilling situation...) and bind them to your desired keybinds or enable them from the side panel.
